### ISSUE TITLE
Every time I open the app, one random playlist does not work.

### ISSUE THREAD

**Issue by InfusedPolse** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3298]
Hi,

Since 4.6.0 or 4.6.1 I've had a problem that I only get when accessing Google Play Music from GPMDP, not in Chrome or any other browser. Randomly, one playlist will work when opening the application. The size of the playlist does not matter. Sometimes, I cannot play or open a playlists with just 2 songs in it and other times it's a playlist with 200+ songs. The songs in the affected playlists can be found a played elsewhere - either in other playlists or in their albums etc.

I have tried uninstalling and reinstalling the application several times but it does not seem to do anything. Not only does this happen but the application itself loads for a lot longer when I open it.

I have attached the debug .zip file to this post but I checked it and it seems like only opening and closing the app has been logged since the latest update (which I do a lot to make certain playlists work) so it will not be of very much help.

**OS: Windows 10**

**GPMDP Version: 4.6.1**

**Issue Descriptions: Random playlists will not work every time I open the app. The app also loads for longer that it usually would.**

**Steps to Reproduce: 
Open up GPMDP and try to open different playlists until you find one that cannot open. 
The application will keep loading and you'll have to close the app and open it again to the able to play any music since you cannot click anywhere. However, if a song/playlist is already playing when you try to access a broken playlist it will keep playing and you can use media keys to skip songs etc. 
Clicking on the play icon on affected playlists will not do anything.**

[debug_info.zip](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/files/2234919/debug_info.zip)

**Comment by dzek69** [comment]
Try clearing app data:

**Comment by InfusedPolse** [comment]
Nope, doesn't work. The first playlist I try to play never plays - not even after playing other playlists. Also, loading times are crazy. Uninstalling and reinstalling 4.6.1 doesn't work but uninstalling and reinstalling 4.5.0 works perfectly :)

**Comment by aaroncoolguy** [comment]
This is still happening to me. Even after a fresh OS install, reinstalling the program, and clearing app data. Sometimes I have seen this happen on a certain artists album however it is usually on playlists and I have noticed that it's ends up being the playlist that I listen to most often or have been listening to a lot more of lately. 
![49ff4006-c24f-41fb-b1f0-c5fb10e592a7 png](https://user-images.githubusercontent.com/46825906/51407390-03006280-1b19-11e9-84e0-9c26f2a9cd7a.jpg)
